cfgov/v1/atomic_elements/organisms.py

```diff
@@ -792,6 +792,29 @@ class FilterableList(BaseExpandable):
                   ' headings in filterable list results'
     )
 
+    filter_children = blocks.BooleanBlock(
+        default=True,
+        required=False,
+        help_text=(
+            "If checked this list will only filter its child pages."
+        ),
+    )
+    filter_siblings = blocks.BooleanBlock(
+        default=False,
+        required=False,
+        help_text=(
+            "If checked this list will only filter its sibling pages."
+        ),
+    )
```

Comment:
These both being booleans with the help text as-is opens the possibility for confusion about what happens if both are checked. The logic in the model checks `filter_children` and then only if that is `False` does it check `filter_siblings`, but that's not evident in the UI created by these block definitions.
1. Should we allow for the possibility of both being true, i.e. a list showing both siblings and children? Grandchildren or further descendants? Nieces and nephews? :trollface: 
2. If not, could we make this a ChoiceBlock defaulting to children?

Reply:
You're right in that the behavior of the booleans is non-obvious in the way they interact. I don't think there's much to do to help that right now – a `ChoiceBlock` might work, but I think it'd add some additional complexity in the implementation that might not be worth it (more on that in a moment). As a general thing, the way archival appears in the admin is already going to be confusing and non-obvious until we actually spend time designing that experience. 

For filterable lists themselves, I think a better way than any of this would be to expose the parent page to filter as a page selector, or, if no page is selected, allow category selection in the admin. But that has a lot more implications that we'd need to handle, and potentially more pages that could break.